packages/frontend/src/app/layout.tsx

```diff
@@ -1,21 +1,23 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import "./globals.css";
+import "../../node_modules/primeflex/primeflex.css";
```

Comment:
Could you change the import to be "/node_modules/primeflex/primeflex.css", it doesn't have to be relative if it's not necessary

Reply:
Could you rewrite it to say 
```jsx
import "/node_modules/primeflex/primeflex.css";
```